docs/platforms/javascript/guides/aws-lambda/install/cjs-layer.mdx

```diff
@@ -0,0 +1,110 @@
+title: Lambda Layer - CJS
+description: "Learn how to add the Sentry Node Lambda Layer to use Sentry in your Lambda functions running in CommonJS (CJS)"
+sidebar_order: 1
+---
+
+The easiest way to get started with Sentry is to use the Sentry [Lambda Layer](https://docs.aws.amazon.com/Lambda/latest/dg/configuration-layers.html) instead of adding `@sentry/aws-serverless` with `npm` or `yarn` [manually](../cjs-manual).
+If you follow this guide, you don't have to worry about deploying Sentry dependencies alongside your function code.
+To actually start the SDK, you can decide between setting up the SDK using environment variables or in your Lambda function code. We recommend using environment variables as it's the easiest way to get started. [Initializing the SDK in code](#alternative-initialize-the-sdk-in-code) instead of setting environment variables gives you more control over the SDK setup if you need it.
+
+<Note>
+
+This installation method **does not** work with Lambda functions running in EcmaScript Modules (ESM) mode, using `import` syntax. If you're running your function in ESM, follow the [ESM guide](../esm-npm).
+
+</Note>
+
+## 1. Prerequisites
+
+Before you begin, make sure you have the following:
+
+- You have a Lambda function that is running in CommonJS (CJS) mode, using `require` syntax.
+- You know the AWS region that your function is deployed to.
+
+## 2. Add the Sentry Lambda Layer
+
+Add the Sentry Layer by navigating to your Lambda function. Select **Layers**, then **Add a Layer**.
+
+![](./img/lambda_view.png)
+
+**Specify an ARN** tab as illustrated:
+
+![](./img/add_layer.png)
+
+Finally, set the region and copy the provided ARN value into the input.
+
+<LambdaLayerDetail canonical="aws-layer:node" />
+
+<br />
+
+## 3. Initialize the SDK with Environment Variables
+
+The easiest way to set up the SDK is to start and configure it using environment variables. This way, you don't have to modify your Lambda function code.
+
+<OnboardingOptionButtons options={["error-monitoring", "performance"]} />
+
+Set the following environment variables in your Lambda function configuration:
+
+```bash {"onboardingOptions": {"performance": "3"}}
+NODE_OPTIONS="-r @sentry/aws-serverless/awslambda-auto"
+SENTRY_DSN="___PUBLIC_DSN___"
+SENTRY_TRACES_SAMPLE_RATE="1.0"
+```
+
+To set environment variables, navigate to your Lambda function, select **Configuration**, then **Environment variables**:
+
+![](./img/cjs_env_vars.png)
+
+## Alternative: Initialize the SDK in Code
+
+Instead of [Step 3, setting environment variables](#3-initialize-the-sdk-with-environment-variables), you can also manually initialize the SDK in your Lambda function code.
+This way, you can customize the SDK setup further.
+Note that you don't have to actually install an NPM package for this to work, as the package is already included in the Lambda Layer.
+
+Make sure you completed [step 1](#1-prerequisites) and [step 2](#2-add-the-sentry-lambda-layer) before proceeding.
```

Comment:
You mean the `--require @sentry/aws-serverless/awslambda-auto` command, right? Not for this (alternative) way in CJS. If users do as suggested in the code snippet, they call both, `Sentry.init` early enough and `Sentry.wrapHandler` around their handler which is what auto-instrumentation would otherwise do for them. 

Unless I'm missing something, this should work w/o env variables.

Reply:
Ah, right. Thanks for confirming. And thanks for the docs writeup. I